aml-development/frontend
# fix(login): prevent accessing login page when logged in

Hotfix to prevent a user from being able to go to the login page (ie `../#/login`) when they are already logged in.  Should work for both certificate-based and username/password authentication.

**Test Scenarios**
1. Log in then hit the back button the browser.  You should not go to the login page and instead go back to where you were before logging in.  This should be true even after visiting some pages, then logging out and logging back in.
2. Log in, then manually change the URL to go to the login page.  You should be redirected to the home page.  Hitting the back/forward button in the browser should not take you to the login page.
